cmd/namespace/validate_test.go

```diff
@@ -0,0 +1,115 @@
+package namespace
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	"github.com/ory/x/cmdx"
+	"github.com/ory/x/configx"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+)
+
+const fileMode = 0644
+
+const (
+	configEmbeddedYaml = `
+dsn: memory
+namespaces:
+  - name: testns0
+    id: 0
+  - name: testns1
+    id: 1
+`
+
+	configEmbeddedJson = `{"dsn":"memory","namespaces":[{"name":"testns0","id":0}]}`
+
+	configEmbeddedToml = `
+[[namespaces]]
+name = "testns0"
+id = 0
+
+[[namespaces]]
+name = "testns1"
+id = 1
+`
+)
+
+const configReference = `
+dsn: memory
+namespaces: %s
+`
+
+const nsyaml = "name: testns0\nid: 0"
+const nsjson = `{"name": "testns1", "id": 1}`
+
+func TestValidateConfigNamespaces(t *testing.T) {
+	cmd := cmdx.CommandExecuter{New: validateCommand}
+
+	t.Run("case=read valid config with embedded namespaces", func(t *testing.T) {
+		dir := t.TempDir()
+		fnyaml := dir + "/keto.yaml"
+		assert.NoError(t, ioutil.WriteFile(fnyaml, []byte(configEmbeddedYaml), fileMode))
```

Comment:
```suggestion
		require.NoError(t, ioutil.WriteFile(fnyaml, []byte(configEmbeddedYaml), fileMode))
```
These are more of a `require`, as it does not make much sense to continue the test if that part fails.

Reply:
Ah, I hadn't encountered the difference before! Java/junit behavior is always `assert`. Will change to require for setup steps.